README.md

```diff
@@ -3,7 +3,7 @@
 ## Installation
 1. Copy the `darkmode.css` file to the corresponding `recipe` (service) folder. For example: below's `[service folder]` means `messenger` or `slack` etc.:
   * Mac: `~/Library/Application Support/Franz/recipes/[service folder]`
-  * Windows: `%appdata%/Franz/recipes/[service folder]`. Press `Windows`+`R` to open the `Run` dialog and paste this path, for example: `%appdata%/Franz/recipes/messenger`
+  * Windows: `%appdata%/Roaming/Franz/recipes/[service folder]`. Press `Windows`+`R` to open the `Run` dialog and paste this path, for example: `%appdata%/Roaming/Franz/recipes/messenger`
```

Comment:
Could you separate this into 2 cases?
1 is that user installs via Franz setup file
2 is that user installs via Chocolately or any other